ml-algorithms/build.gradle

```diff
@@ -85,8 +85,7 @@ dependencies {
     implementation('net.minidev:json-smart:2.5.2')
     implementation group: 'org.json', name: 'json', version: '20231013'
     implementation group: 'software.amazon.awssdk', name: 'netty-nio-client', version: "2.30.18"
-    // TODO: Replace the custom jar with dependency implementation 'io.modelcontextprotocol.sdk:mcp:0.10.0' after its release
-    api files('libs/mcp-0.10.0-SNAPSHOT.jar')
```

Comment:
Shall we remove the libs folder and this jar then?

https://github.com/opensearch-project/ml-commons/blob/main/ml-algorithms/libs/mcp-0.10.0-SNAPSHOT.jar

Reply:
We need to use 0.9.0 instead of 0.10.0 jar.

Reply:
It makes sense to remove the libs folder as we don't need it now.

Reply:
Since he is already pulling dep from maven there is no reason to have the libs folder at local with a 0.10 version of the jar.